aleph-client/src/debug/treasury.rs

```diff
@@ -28,7 +27,7 @@ pub fn print_storage<C: AnyConnection>(connection: &C) {
     println!("{}", entry_prompt("Proposals"));
     for x in 0..=proposal_count {
         let p: Option<Proposal<AccountId32, Balance>> = connection
-            .get_storage_map("Treasury", "Proposals", x, None)
+            .read_storage_map("Treasury", "Proposals", x, None)
             .unwrap();
```

Comment:
`read_storage_map` unwraps from `ApiResult`, then this unwrap should unwrap from `Option`, so the ascription should be rather `Proposal<AccountId32, Balance>`

Reply:
Changed.